lib/guardian/tag_guardian.rb

```diff
@@ -8,7 +8,10 @@ def can_see_tag?(_tag)
 
   def can_create_tag?
     SiteSetting.tagging_enabled &&
-      @user.has_trust_level_or_staff?(SiteSetting.min_trust_to_create_tag)
+      (
+        @user.has_trust_level_or_staff?(SiteSetting.min_trust_to_create_tag) ||
```

Comment:
Since this is core's code, why do we still reference the deprecated site setting here?

Reply:
From the PR description I think he is trying to maintain backwards compatibility with plugins until they are all updated.

Reply:
Yes this is only necessary very occasionally for settings that are more widely used and is only temporary. Many of these settings moves only needed core changes.